firmware/lib/config/config.h.template

```diff
@@ -3,18 +3,21 @@
 
 
 // ============= CONFIGURE THESE FIELDS BEFORE FLASHING ====================================================
-#define WIFI_SSID "WIFINAME" // Wifi name (please use 2.4gz network)
-#define WIFI_PASS "WIFIPASS" // Wifi password
+#define WIFI_SSID "WIFINAME" // wifi name (please use 2.4gz network)
+#define WIFI_PASS "WIFIPASS" // wifi password
```

Comment:
I am not sure we need the capitalization changes but if we do make them they should be in a separate PR. As they do not relate to adding german and french support.